torchci/rockset/metrics/__sql/queued_jobs_by_label.sql

```diff
@@ -48,6 +48,7 @@ SELECT
   COUNT(*) AS count,
   MAX(queue_s) AS avg_queue_s,
   machine_type,
+  CURRENT_TIMESTAMP() AS _event_time
```

Comment:
Maybe a comment on why we need to use CURRENT_TIMESTAMP as _event_time here.  I guess that this is because Rockset uses the time when it ingests the data as _event_time.  So, re-importing records from S3 will set them all to the same _event_time?

Reply:
Yes _event_time is the time records get added to rockset if it is empty https://docs.rockset.com/documentation/docs/special-fields#the-_event_time-field

I decided to make a different field entirely so theres less confusion between the two